src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -94,8 +96,10 @@ const DeleteOrgButton: FC = () => {
         dispatch(notify(deleteOrgWarning(buttonElement)))
       } else {
         dispatch(
-          showOverlay('suspend-org-in-paid-account', null, () =>
-            dispatch(dismissOverlay())
+          showOverlay(
+            'suspend-org-in-paid-account',
+            {users: users.length},
+            () => dispatch(dismissOverlay())
```

Comment:
I would handle the situation where `users` isn't set here or in the child component. `users` is coming from an API call that's made in the context, so I believe the page can load before users comes back (and before the user tries to open the overlay).

One way would be: 
`{users: users.length || 0}`